src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.lib.DaoBook;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
```

Comment:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
```
stick to the constant naming convention